src/components/shared/title-animation/title-animation.jsx

```diff
@@ -77,15 +77,17 @@ const TitleAnimation = ({ className, tag: Tag, items, animationName, controls })
     <>
       <Tag className={clsx(animationName === 'first' && 'md:hidden', className)}>
         {itemsWithAnimationData.map(({ value, color, delay }, index) => (
-          <motion.span
-            initial="initial"
-            animate={controls}
-            variants={animations[animationName]}
-            custom={{ color, delay }}
-            key={index}
-          >
-            {value}{' '}
-          </motion.span>
+          <LazyMotion features={domAnimation}>
```

Comment:
It might be better to wrap the entire element  <Tag></> in <LazyMotion features={domAnimation}></LazyMotion> because now there might be problems with <li> — they might lost key